### PR TITLE
Use separate admin layout

### PR DESCRIPTION
**Pull request by creuter**
This adds nested layouts, so the admin pages don't have to share the nav and wrapping container from the public side of things.

It also adds a new, separate site header for admin pages, and changes the content of the public footer.

screen shot 2017-09-26 at 1 43 31 pm

screen shot 2017-09-26 at 1 43 49 pm

### CODE REVIEW

File: web/web.ex [hunk 1]
```diff
@@ -43,6 +43,19 @@ defmodule Community.Web do
     quote do
       use Phoenix.View, root: "web/templates"
 
+      def render_embeddable(module, template, assigns) do
```
Comment (creuter) [inline web/web.ex:46]: Draper gave a 👍 IRL to this stuff.

File: web/static/css/_site-navigation.scss [hunk 2]
```diff
@@ -1,27 +1,63 @@
...
-  a {
-    color: $lighter-font-color;
-    display: inline-block;
-    font-size: $small-font-size;
-    font-weight: 600;
+.navigation-link {
+  color: $light-font-color;
+  display: block;
+  font-size: $small-font-size;
+  font-weight: 600;
```
Comment (brendastorer) [inline web/static/css/_site-navigation.scss:33]: Do you have font weights declared in variables? Is it worth it?

**Review by brendastorer — COMMENTED** [review]
Teeny tiny question, but it all looks awesome!

**Comment by drapergeek** [comment]
LGTM

File: web/static/css/_site-navigation.scss (hunk 2 above)
Comment (creuter) [reply web/static/css/_site-navigation.scss:33]: I don't think it's worth it. They're descriptive, and I've found it rare that they need to change if/when the typeface changes.

**Pull request merged**